ci/builders/README.md

```diff
@@ -374,9 +374,126 @@ be relative to the checkout directory.
 
 ### Global Tests
 
-Global tests follow the same format as build tests but defined at the top
-level. The main difference from local tests is that global tests have access
-to the sub-build artifacts.
+Tests in this section run on a separate bot as an independent sub-builds.
+As opposed to tests running within builds, global tests have access to the
+the outputs of all the builds running in the same orchestrator build. A
+use case for global tests is to run flutter/framework tests using the
+artifacts generated by an specific engine build.
+
+Global tests currently support two different scenarios:
+
+* flutter/flutter tests with [tester](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester.py)
+  recipe. This workflow checks out flutter/flutter to run any of the existing
+  sharded tests using the engine artifacts archived to GCS.
+* complicated engine tests that require the outputs from multiple subbuilds
+  with [tester_engine](https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/engine_v2/tester_engine.py).
+  This workflow checks out [flutter/engine] and operates over the dependencies passed to it using cas.
```

Comment:
You might not like this but it may be good to list out the benefits or what each of these different recipes gets you. Like one provides a checkout of flutter/flutter, etc...

Reply:
That could be done in the future.

Reply:
Sounds good to me.